oxen-rust/src/lib/src/repositories/tree.rs

```diff
@@ -170,10 +219,10 @@ pub fn get_node_by_path(
                 }
             }
         }
-        _ => match CommitMerkleTreeLatest::from_path(repo, commit, path, load_recursive) {
-            Ok(tree) => Ok(Some(tree.root)),
+        _ => match CommitMerkleTreeLatest::read_from_path(repo, commit, path, load_recursive) {
+            Ok(node) => Ok(node),
             Err(e) => {
-                log::warn!("Error getting node by path: {e:?}");
+                eprintln!("Error getting node by path: {e:?}");
```

Comment:
We generally shouldn't use `eprintln` from lib code, I think `log::warn` is better here.